### PR TITLE
Create homepage

### LINKED ISSUE #23: Add '/' route to API
The API needs a homepage that provides documentation on how to use the API. This information can include endpoints, params, etc. This does need to be updated as the API gets updated. 

- [x] Create '/' route
- [x] Test to ensure it returns a 200 status code 
- [x] Add what information we currently have on endpoints

### PR DESCRIPTION
**Pull request by linda-le1**
## What functionality does this accomplish?
closes #23 

**Description:**
This adds a homepage with some text at '/'. I've also added something like simplecov called coverage(https://coverage.readthedocs.io/en/coverage-5.0.4/) to the requirements.txt and in the travis.yml, so we can track test coverage. 

## What did you struggle on to complete?
Had to check the difference between coverage html (the output of this was cleaner) and coverage report in the travis yml. I also had to squash some commits, which I've never done before. 

### Test Coverage Percentage: x%
- [ ] No Tests have been changed
- [X] New Tests have been implemented
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened):

## Checklist:
- [X] My code has no unused/commented out code
- [X] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have fully tested my code
- [ ] I have partially tested my code (please explain why):

## Helpful Resources:
* Resource link AND small description of info gathered/learned
https://coverage.readthedocs.io/en/coverage-5.0.4/ (general coverage info)
https://coverage.readthedocs.io/en/v4.5.x/cmd.html (how to use in command line)

## Please include a gif of how you feel about this branch:
giphy

### CODE REVIEW

**Comment by freeheeling** [comment]
Great thought to add a test coverage tool. I'll be sure to reference the resources you included in case I need to install anything locally and for guidance on how to utilize the tool.

**Review by freeheeling — APPROVED** [review]
Approved.

File: .travis.yml [hunk 1]
```diff
@@ -6,7 +6,9 @@ before_install:
 install:
 - pip install -r requirements.txt
 script:
-- pytest
+- coverage erase
+- coverage run -m pytest && coverage html
```
Comment (Jonpatt92) [inline .travis.yml:10]: Nice setup for Travis

File: base_app/base.py [hunk 2]
```diff
@@ -10,6 +10,10 @@
 
 app = Flask(__name__)
 
+@app.route('/', methods=['GET'])
+def home():
+    return "<h1>Welcome to the Hangry-8Ball API!</h1><p>Enter API documentation and endpoints here.</p>"
```
Comment (Jonpatt92) [inline base_app/base.py:15]: This is a great placeholder. We can start updating this with new features after this initial set is complete.

**Review by Jonpatt92 — APPROVED** [review]
Thanks for implementing a testing coverage tracker

**Pull request merged**